src/fields/Resources/ResourcesItem/resources-item.scss

```diff
@@ -64,10 +65,10 @@
     background-color: $warm-grey;
     color: $black;
     display: inline-block;
-    font-size: 13px;
-    line-height: 16px;
-    margin-right: 10px;
-    padding: 0 5px;
+    font-size: 14px;
+    line-height: 1;
+    margin-right: 5px;
```

Comment:
Magische pixel waardes allemaal. Is het misschien af te leiden uit variabelen?

Reply:
Overgenomen uit Atlas